fluent/src/parser.js

```diff
@@ -131,32 +131,36 @@ class RuntimeParser {
       this._index++;
 
       this.skipInlineWS();
-      this.skipBlankLines();
-      this.skipInlineWS();
 
-      val = this.getPattern();
+      if (this._source[this._index] === '\n') {
+        this.skipBlankLines();
+        if (this._source[this._index] === ' ') {
+          this.skipInlineWS();
+          val = this.getPattern();
+        }
+      } else {
+        val = this.getPattern();
```

Comment:
Is this `else` block an optimization for the most common use-case of a one-line `key = value`? If so, can you add a comment explaining it, please?